PaulasWeatherApp/PaulasWeatherApp/ViewController/CurrentWeatherViewController.swift

```diff
@@ -8,11 +8,108 @@
+        tempLabel.textColor = .black
+        tempLabel.text = "... °C"
+        tempLabel.backgroundColor = .clear
+        tempLabel.sizeToFit()
+        
+        view.addSubview(tempLabel)
+        
+        NSLayoutConstraint.activate([
+            tempLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            tempLabel.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: -70),
+        ])
+    }
+    
+    private func setSunsetLabel() {
+        sunsetLabel.translatesAutoresizingMaskIntoConstraints = false
+        sunsetLabel.textAlignment = .center
+        sunsetLabel.font = .systemFont(ofSize: 12)
+        sunsetLabel.textColor = .black
+        sunsetLabel.backgroundColor = .clear
+        sunsetLabel.attributedText = sunsetString
+        sunsetLabel.sizeToFit()
+        
+        view.addSubview(sunsetLabel)
+        
+        NSLayoutConstraint.activate([
+            sunsetLabel.leadingAnchor.constraint(equalTo: sunriseLabel.trailingAnchor,constant: 10),
+            sunsetLabel.trailingAnchor.constraint(equalTo: tempLabel.trailingAnchor),
+            sunsetLabel.topAnchor.constraint(equalTo: tempLabel.bottomAnchor)
+        ])
+    }
+    
+    private func setSunriseLabel() {
+        sunriseLabel.translatesAutoresizingMaskIntoConstraints = false
+        sunriseLabel.textAlignment = .center
+        sunriseLabel.font = .systemFont(ofSize: 12)
+        sunriseLabel.textColor = .black
+        sunriseLabel.backgroundColor = .clear
+        sunriseLabel.attributedText = sunriseString
+        sunriseLabel.sizeToFit()
+        view.addSubview(sunriseLabel)
+        
+        NSLayoutConstraint.activate([
+            sunriseLabel.leadingAnchor.constraint(equalTo: tempLabel.leadingAnchor),
+            sunriseLabel.topAnchor.constraint(equalTo: tempLabel.bottomAnchor)
+        ])
+    }
+    
+     func setCurrentWeatherData(_ data : WeatherDataModel) {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "hh:mm a"
+        
+        tempLabel.text = "\(data.current.temp)  °C"
+        guard let sunset = data.current.sunset, let sunrise = data.current.sunrise  else { return }
+        
+        sunsetString.append(NSAttributedString(string: dateFormatter.string(from: Date(timeIntervalSince1970: Double(sunset))) ))
+        sunsetLabel.attributedText = sunsetString
+        
+        sunriseString.append(NSAttributedString(string: dateFormatter.string(from: Date(timeIntervalSince1970:Double(sunrise)))))
+        sunriseLabel.attributedText = sunriseString
+    }
+    
+    private func createString(for name: String) ->NSMutableAttributedString {
+        let iconsSize = CGRect(x: 0, y: -5, width: 15, height: 15)
+        let imageAttachment = NSTextAttachment()
+        imageAttachment.image = UIImage(systemName: name)
+        imageAttachment.bounds = iconsSize
+        let atr = NSMutableAttributedString(attachment: imageAttachment)
+        return atr
```

Comment:
Minor: we have another rule defined in the project, that the return is separated by an empty line from the other code.